Header header

obj_info2[] objects
sensor_msgs/Image image
int16 type